source/index.js

```diff
@@ -15,6 +15,7 @@ export { default as aperture } from './aperture';
 export { default as append } from './append';
 export { default as apply } from './apply';
 export { default as applySpec } from './applySpec';
+export { default as applyTo } from './applyTo';
```

Comment:
Is this something we will need to add into our PRs in the future? Can we get a build script to do this?

Reply:
I was in favor of doing this automatically via a script when we first added an index.js file but we decided against it.. I think it is fine to do it manually as I believe in all cases (adding, moving, deleting functions) either tests or building will fail if index.js is not updated.

Reply:
It may be worth it at this point to update the instructions in CONTRIBUTING.md